deploy/template/prometheus-rule.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+    role: alert-rules
+  name: application-monitoring
+  namespace: [[ .Namespace ]]
+spec:
+  groups:
+    - name: general.rules
+      rules:
+        - alert: KeycloakJavaHeapThresholdExceeded
+          annotations:
+            message: >-
+              {{ printf "%0.0f" $value }}% heap usage of {{ $labels.area }} in pod {{
+              $labels.pod }}, namespace {{ $labels.namespace }}.
+          expr: |
+            100 * jvm_memory_bytes_used{area="heap",namespace="[[ .Namespace ]]"}
+              / jvm_memory_bytes_max{area="heap",namespace="[[ .Namespace ]]"}
+              > 90
+          for: 1m
+          labels:
+            severity: warning
```

Comment:
@pb82 Based on this comment I think these should all be critical severity (https://github.com/integr8ly/installation/pull/333#issuecomment-461468700) ?

Reply:
good point, will update